client/src/components/InlinePanel/index.js

```diff
@@ -209,6 +209,36 @@ export class InlinePanel extends ExpandingFormset {
     );
   }
 
+  addTabIndex(prefix) {
```

Comment:
Can we call this method `initialFocus`

```suggestion
  initialFocus(prefix) {
```